website/docs/compute/local-deployment/local-deployment.md

```diff
@@ -0,0 +1,219 @@
+# Local Deployment
```

Comment:
This might sound as nitpicky, but technically you could have a whole *Kubernetes* cluster set up locally and be deploying to it (or even, `docker compose` is also "Local Deployment") so I'd add "with Minikube" or something to differentiate (also :money_mouth_face: SEO :moneybag: )

Reply:
"with Minikube" is awesome, thank you for the tip!